notebooks/tfx_pipelines/walkthrough/solutions/tfx_walkthrough_vertex.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    )
</pre><p>Consider adding a component name .with_id("csv_examplegen") as a best practice for each component as a new enhancement; this is what is visualized in DAG / on Vertex Pipelines. I believe this style is new to TFX &gt; 1.0.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done!</p>